dockerfile

```
@@ -0,0 +1,34 @@
+FROM python:3.10-slim-bullseye AS build 
+RUN apt-get update
+RUN apt-get install -y curl
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+ENV POETRY_HOME="/opt/poetry" 
+ENV PATH="${PATH}:/root/.poetry/bin"
+RUN mkdir /opt/todoapp/
+WORKDIR /opt/todoapp/
+COPY poetry.toml /opt/todoapp/
+COPY pyproject.toml /opt/todoapp/
+COPY poetry.lock /opt/todoapp/
+
+EXPOSE 8000
+ENV EnvironmentFile=/opt/todoapp/.env
+
+FROM build AS prod
+
+COPY todo_app/. /opt/todoapp/todo_app
```

I don't think the extra `.` makes any difference, but it's a bit confusing
```suggestion
COPY todo_app/ /opt/todoapp/todo_app
```